cnf-certification-test/performance/suite.go

```diff
@@ -33,6 +33,10 @@ import (
 	"github.com/test-network-function/cnf-certification-test/pkg/tnf"
 )
 
+const (
+	LimitedUseOfExecProbes = 10
```

Comment:
add separate parameters for minimum period in seconds and maximum number of exec probes per CNF, LimitedUseOfExecProbes seems to cover both